### PR TITLE
chore: Fix spelling errors

### PR DESCRIPTION
**Pull request by candrews**
Spell checking done using codespell.

Correct spelling is a great way to look more professional and demonstrate higher quality. And it catches a surprisingly number of actual errors, too.

Following up on https://github.com/aws-controllers-k8s/iam-controller/pull/78

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by candrews** [comment]
> Probably best to ignore anything under `pkg/testdata` as this is bulk imported from an external source.

Done! :+1:

**Review by jaypipes — APPROVED** [review]
Awesome, thank you @candrews! :)

File: pkg/testdata/models/apis/codedeploy/0000-00-00/docs-2.json [hunk 1]
```diff
@@ -45,7 +45,7 @@
     "SkipWaitTimeForInstanceTermination": "<p>In a blue/green deployment, overrides any specified wait time and starts terminating instances immediately after the traffic routing is complete.</p>",
```
Comment (RedbackThomson) [inline pkg/testdata/models/apis/codedeploy/0000-00-00/docs-2.json:45]: Looks like you're actually finding spelling mistakes in official AWS models. Sounds like we need this tool to run internally, too. 😆

**Review by RedbackThomson — APPROVED** [review]

**Pull request merged**